Makefile

```
@@ -25,11 +25,19 @@ dry-run: build
 
 .PHONY: image
 image: build
-	docker build -t quay.io/rhacs-eng/qa:cve-uploader-$(TAG) image
+	docker build -t quay.io/rhacs-eng/cve-uploader:$(TAG) image
 
 .PHONY: push
 push: image
-	docker push quay.io/rhacs-eng/qa:cve-uploader-$(TAG) | cat
+	docker push quay.io/rhacs-eng/cve-uploader:$(TAG) | cat
```

It looks like `| cat` might be mangling `docker push` exit status and so the step seems like it succeeds whereas it should actually fail.
See https://github.com/stackrox/k8s-istio-cve-pusher/actions/runs/3389106193/jobs/5631834484#step:8:42
```
docker push quay.io/rhacs-eng/cve-uploader:0.1.2-8-ge63be8fffd | cat
The push refers to repository [quay.io/rhacs-eng/cve-uploader]
6b6235813b95: Preparing
e7bbfd3ff7a9: Preparing
994393dc58e7: Preparing
unauthorized: access to the requested resource is not authorized
```

I think we used `| cat` with `docker pull/push` when it was CircleCI of earlier versions. I think it may be unnecessary in GitHub Actions.

One option is to drop `| cat`
```suggestion
	docker push quay.io/rhacs-eng/cve-uploader:$(TAG)
```

Another option might be to configure shell in this Makefile with `pipefail` similar to how it's done for Operator.
https://github.com/stackrox/stackrox/blob/d89f45b3a2ff1207e960a542bf260fd56ac5f7f2/operator/Makefile#L111-L112

----

Apart from this, I added write permissions to `QUAY_RHACS_ENG_RW` account so these commands should succeed now.

---

I'll just drop the `cat`, and thank you :)